store/sqlstore/channel_store.go

```diff
@@ -3503,6 +3675,31 @@ func (s SqlChannelStore) SearchGroupChannels(userId, term string) (model.Channel
 	return groupChannels, nil
 }
 
+func (s SqlChannelStore) SquirrelSearchGroupChannels(userId, term string) (model.ChannelList, error) {
+	isPostgreSQL := s.DriverName() == model.DatabaseDriverPostgres
+	query := s.squirrelGetSearchGroupChannelsQuery(userId, term, isPostgreSQL)
+
+	sql, params, err := query.ToSql()
+	if err != nil {
+		mlog.Error(sql)
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.